packages/components/src/components/hds/app-side-nav/toggle-button.hbs

```diff
@@ -0,0 +1,7 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<button class="hds-app-side-nav__toggle-button" type="button" ...attributes>
```

Comment:
Issue, blocker: this seems to be missing an accessible name.

Reply:
Additional issue, blocker: when I have VoiceOver open and I toggle the menu open or closed, there is no `aria-expanded` attribute telling me if it's open or closed.

Reply:
I will check into these, thanks.

Reply:
In index.ts there is code which adds an ariaLabel to the toggleButton:

```
get toggleButtonAriaLabel(): string {
    if (this.isMinimized) {
      return this.args.toggleButtonAriaLabel ?? 'Open menu';
    }
    return this.args.toggleButtonAriaLabel ?? 'Close menu';
  }
```

I had already added an item to the list of follow-on tasks (in PR description) to improve this (partially by adding aria-expanded as mentioned in linked comment from @shleewhite ). It looks like you implemented the above in the original SideNav code btw.

Reply:
- PR Branch: https://github.com/hashicorp/design-system/pull/2463/commits/43e57208bec1381d33b4ecb98438b9e63e567ae4
- Commit: https://github.com/hashicorp/design-system/pull/2463/commits/43e57208bec1381d33b4ecb98438b9e63e567ae4